01_P/P_2_3_4_01/sketch.js

```diff
@@ -53,10 +54,9 @@ function preload() {
 }
 
 function setup() {
-  // use full screen size 
+  // use full screen size
   createCanvas(displayWidth, displayHeight);
   background(255);
-  smooth();
```

Comment:
Removed `smooth()` since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).